.github/workflows/backend_plugin_test.yml

```diff
@@ -0,0 +1,58 @@
+name: Backend Plugin Testing
+
+on:
+  pull_request:
```

Comment:
This means tests will run in Github Actions when:
- a pull request is issued to the main branch; or,
- on all pushes.

The former scenario will enable us to require passing status checks on merges to main, while the latter will enable us to see the state of these tests on pushes (likely, on pushes to our own forks while we're working on development).

The outcome of these tests isn't visible (yet) from the PR itself since this PR - which includes this workflow - has not been merged yet.  However, you can see the state of these tests (passing) in my local fork: https://github.com/lorawoodford/lassb-ead-exporter/actions/runs/10567256452/job/29275830388